### DISCUSSION TITLE
Microsof Sign in fix?

### DISCUSSION THREAD

**Discussion by cardonadon** [Authenticator-Extension/Authenticator#626] [General]
<You can also suggest feedback by Tweeting with #AuthenticatorFeedck>
I'm having trouble resetting my MS 
**Describe the feature you want:**
I'm having trouble getting into Ms account. My Authenticator

**Why do you want this feature in Authenticator?**
i dont know.....get it working again

**Comment by mymindstorm** [comment]
Please provide more details and reproduction steps if possible.
